### PR TITLE
Upgrade the Gradle wrapper to 7.5.1

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: gradle.lockfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This is a Gradle generated file for dependency locking.
+# Manual edits can break the build and are not advised.
+# This file is expected to be part of source control.
+org.embulk:embulk-api:0.10.36=compileClasspath
+org.embulk:embulk-util-rubytime:0.3.2=runtimeClasspath
+org.embulk:embulk-util-timestamp:0.2.1=compileClasspath,runtimeClasspath
+org.msgpack:msgpack-core:0.8.11=compileClasspath
+org.slf4j:slf4j-api:1.7.30=compileClasspath
+empty=
```
Comment (vietnguyen-td) [inline gradle.lockfile:9]: this is new format when using gradle-7.5.1?

**Review by vietnguyen-td — APPROVED** [review]
LGTM

File: gradle.lockfile (hunk 1 above)
Comment (dmikurube) [reply gradle.lockfile:9]: Yes, it's new from Gradle 7: https://gradle.org/whats-new/gradle-7/#single-dependency-lock-file-format

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**